app/assets/stylesheets/components/_theme.scss

```diff
@@ -121,3 +121,17 @@ html{
   height: 100%;
   text-rendering: optimizeLegibility;
 }
+
+select {
```

Comment:
How come a reset does so much and even sets such specific styles such as width and height